data/map planets.txt

```diff
@@ -347,6 +347,7 @@ planet "Blue Interior"
 planet Bluerun
 	attributes hai station uninhabited
 	landscape land/station33
+	music ambient/machinery
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
```

Comment:
I'm not sure this description suggests we should hear much from this station, certainly not more noise than our own ship might make.